product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2015 OdooMRP team
+# Copyright 2015 AvanzOSC
+# Copyright 2015 Tecnativa
+# Copyright 2018 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+
+
+class TestProductSupplierinfoForCustomer(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
```

Comment:
@marielejeune Couild you inherit from BaseCommon to avoid this? See: https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L21